clickhouse/config.xml

```diff
@@ -1,10 +1,6 @@
 <yandex>
-    <max_server_memory_usage_to_ram_ratio>
-        <!-- This include is important!
-         It is required for the version of Clickhouse
-         used on ARM to read the environment variable. -->
-        <include from_env="MAX_MEMORY_USAGE_RATIO"/>
-    </max_server_memory_usage_to_ram_ratio>
+ <!-- This include is important! It is required for the version of Clickhouse used on ARM to read the environment variable. -->
```

Comment:
For some reason, in more recent versions of clickhouse the newlines here are causing parsing issues, so I just removed them